build/controllers/ClassmapController.php

```diff
@@ -60,7 +60,7 @@ public function actionCreate($root = null, $mapFile = null)
             if (strpos($file, $root) !== 0) {
                 throw new Exception("Something wrong: $file\n");
             }
-            $path = str_replace('\\', '/', substr($file, strlen($root)));
+            $path = str_replace('\\', '/', substr($file, \strlen($root)));
```

Comment:
Why not `substr` and `str_replace`?

Reply:
They don't have opcodes, the PR addresses only functions which gets opcodes generated (hence can contribute to better performance). Here the reference which functions covered: https://github.com/php/php-src/blob/f2db305fa4e9bd7d04d567822687ec714aedcdb5/Zend/zend_compile.c#L3872